packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -2,18 +2,58 @@
 const expect = require('expect');
 const { accessibility } = require('../commands/validates');
 const { toBeAccessible } = require('../commands/expect');
+const viewportHelpers = require('../commands/viewport-helpers');
+const hideInputCaret = require('../commands/hide-input-caret');
 
 class TerraService {
   /**
    * Service hook executed prior to test execution.
    * Initializes the Terra Service's custom commands.
    */
-  before() {
+  before(capabilities) {
+    /* Add the Jest expect module the use the Jest matchers. */
     global.expect = expect;
     global.expect.extend({ toBeAccessible });
 
-    global.Terra = {};
-    global.Terra.validates = { accessibility };
+    /* Add a Terra global with access to Mocha-Chai test helpers. */
+    global.Terra = {
+      validates: { accessibility },
+
+      /* `viewports` provides access Terra's list of test viewports. */
```

Comment:
nit: Do the back-ticks being used here do/mean something?

Reply:
No, they just add more emphasis to these key names. They are now removed. https://github.com/cerner/terra-toolkit/pull/438/commits/19c190061a6d0f428d0ab4b33b669bed04d9f219